src/components/IncompatibleEditorDevice.tsx

```diff
@@ -58,8 +58,9 @@ const IncompatibleEditorDevice = ({
                     values={{
                       link: (children) => (
                         <Link
-                          href="https://support.microbit.org/support/solutions/articles/19000154234-which-version-of-micro-bit-do-i-have-"
                           color="brand.600"
+                          textDecoration="underline"
+                          href="https://support.microbit.org/support/solutions/articles/19000154234-which-version-of-micro-bit-do-i-have-"
```

Comment:
Not related to this PR, but this should presumably also be an external link with the target and rel props.

Reply:
Makes sense [fa86dc1](https://github.com/microbit-foundation/ml-trainer/pull/492/commits/fa86dc143e5a7dea3a63f1e3272979940ae2202b)